Vic2ToHoI4/Data_Files/configurables/cultureGroupToGraphics.txt

```diff
@@ -4137,29 +5439,92 @@ deathbringers = {
 		}
 	}
 	male_operative_portraits = {
-		"gfx/leaders/Deathbringers/portrait_synthetic_operative.dds"
+		"gfx/leaders/Animalia/portrait_Spymaster_Glitterhoof.dds"
+		"gfx/leaders/Animalia/portrait_unicorn_disguise_buisness.dds"
 	}
 	female_operative_portraits = {
-		"gfx/leaders/Deathbringers/portrait_synthetic_operative.dds"
+		"gfx/leaders/Animalia/portrait_Spymaster_Glitterhoof.dds" # Revise
 	}
 	male_monarch_portraits = {
-		"gfx/interface/ideas/eldritch.dds"
+		"gfx/interface/ideas/idea_porsche.dds"
 	}
 	female_monarch_portraits = {
-		"gfx/interface/ideas/eldritch.dds"
+		"gfx/interface/ideas/idea_porsche.dds"
 	}
 	graphical_culture = western_european_gfx
 	graphical_culture_2d = western_european_2d
 }
 
+komodo = {
+	cultures_and_groups = { dragon_culture }
+	
+	army_portraits = {
+		"gfx/leaders/Animalia/portrait_dragon_businessman_leader.dds"
+	}
+	navy_portraits = {
+		"gfx/leaders/Animalia/portrait_dragon_businessman_leader.dds"
+	}
+	leader_portraits = {
+		communism = {
+			"gfx/leaders/Animalia/portrait_dragon_businessman_leader.dds"
+		}
+		democratic = {
+			"gfx/leaders/Animalia/portrait_dragon_businessman_leader.dds"
+		}
+		fascism = {
+			"gfx/leaders/Animalia/portrait_dragon_businessman_leader.dds"
+		}
+		absolutist = {
+			"gfx/leaders/Animalia/portrait_dragon_businessman_leader.dds"
+		}
+		neutrality = {
+			"gfx/leaders/Animalia/portrait_dragon_businessman_leader.dds"
+		}
+		radical = {
+			"gfx/leaders/Animalia/portrait_dragon_businessman_leader.dds"
+		}
+	}
+	ideology_minister_portraits = {
+		communism = {
+			"gfx/interface/ideas/spr_anarchist_society.dds"
+		}
+		democratic = {
+			"gfx/interface/ideas/spr_anarchist_society.dds"
+		}
+		fascism = {
+			"gfx/interface/ideas/spr_anarchist_society.dds"
+		}
+		absolutist = {
+			"gfx/interface/ideas/spr_anarchist_society.dds"
+		}
+		neutrality = {
+			"gfx/interface/ideas/spr_anarchist_society.dds"
+		}
+		radical = {
+			"gfx/interface/ideas/spr_anarchist_society.dds"
+		}
+	}
+	male_operative_portraits = {
+		"gfx/leaders/Animalia/portrait_dragon_businessman_operative.dds"
+	}
+	female_operative_portraits = {
+		"gfx/leaders/Animalia/portrait_dragon_businessman_operative.dds" # Revise
+	}
+	male_monarch_portraits = {
+		"gfx/interface/ideas/idea_man_low_legitimacy5.dds"
+	}
+	female_monarch_portraits = {
+		"gfx/interface/ideas/idea_man_low_legitimacy5.dds"
+	}
+	graphical_culture = western_european_gfx
+	graphical_culture_2d = asian_2d
+}
+
 animalia = {
```

Comment:
I'd split the Animalia into single cultures, as dogs ruling a dragon kingdom would be weird and we can now avoid that